devtools/conda-envs/test.yaml

```diff
@@ -25,6 +25,9 @@ dependencies:
   - pytorch
   - rdkit
   - seaborn
+  - boto3
+  - moto
+  - click
 
```

Comment:
Does this also need the `flask` stuff in `pyproject.toml` or is this pulled in by something else.

Reply:
Added! In general we want to put as much in the `conda` from `conda-forge` as we can, and the remainder will get pulled in via `pip` on install of the subpackage.